parlai/scripts/train_model.py

```diff
@@ -475,12 +514,16 @@ def validate(self):
             self.agent.receive_metrics(valid_report)
 
         # check which metric to look at
-        if '/' in opt['validation_metric']:
-            # if you are multitasking and want your validation metric to be
-            # a metric specific to a subtask, specify your validation metric
-            # as -vmt subtask/metric
-            subtask = opt['validation_metric'].split('/')[0]
-            validation_metric = opt['validation_metric'].split('/')[1]
+        if 'tasks' in valid_report:
+            if '/' in opt['validation_metric']:
+                # if you are multitasking and want your validation metric to be
+                # a metric specific to a subtask, specify your validation metric
+                # as -vmt subtask/metric
+                subtask = opt['validation_metric'].split('/')[0]
+                validation_metric = opt['validation_metric'].split('/')[1]
+            else:
+                subtask = 'all'
```

Comment:
Previously the meta-averages ended up in the root report, not in a sub dictionary. where do they end up now? is the report gonna end up being a dictionary with exactly one key ('tasks')?

One could just as easily use `--validation-metric all/metric` i guess?

Reply:
i can put it in the root. i guess i wanted to make the distinction clear that this is really an average across tasks, and not a true per-example measure. my solution (putting them in `all`) isn't great either. maybe we can discuss in person